internal/proxy/task.go

```diff
@@ -1081,6 +1082,29 @@ func (t *alterCollectionTask) PreExecute(ctx context.Context) error {
 		}
 	}
 
+	_, ok := common.IsReplicateEnabled(t.Properties)
+	if ok {
+		return merr.WrapErrParameterInvalidMsg("can't set the replicate.id property")
+	}
+	endTS, ok := common.GetReplicateEndTS(t.Properties)
+	if ok && collBasicInfo.replicateID != "" {
```

Comment:
add a param at AllocTimestampRequest and block wait at rootcoord.

So we don't need to sleep here